### DISCUSSION TITLE
the installer freezes the live iso

### DISCUSSION THREAD

**Discussion by Ccalibur** [helloSystem/hello#516] [Q&A]
when trying to install hellosystem when i click on install hellosysteme ever thing freezes and on the spinning cursor is showing 

am trying that on a late 2012 Mac mini with an intel core i5 3210m intel hd4000 and 10gb of ram 500gb hdd![IMG_20230820_170219.jpg](https://github.com/helloSystem/hello/assets/79602474/0f2e9820-0801-4e2e-b14e-5ce3a78cdae4)

**Comment by probonopd** [comment]
Which exact ISO are you using? When exactly does this happen (what is the last thing you see from the installer)?
Thanks.
